src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.ts

```diff
@@ -89,8 +89,8 @@ export class WorkItemDetailComponent implements OnInit {
   getAllUsers() {
     this.userService.getAllUsers()
       .then((users) => {
-        this.users = users;
-        this.filteredUsers = users;
+        Object.assign(this.users, users);
```

Comment:
Did you know that this is only a shallow copy? It works for now but if the User has any children they will be references not copies.